static/cfg-view.js

```diff
@@ -156,6 +157,16 @@ define(function (require) {
             this.saveState();
         }
     };
+    
+    Cfg.prototype.closeCfg = function () {
+         _.defer(function (self) {
```

Comment:
Why the defer?

This can be rewritten to be _.defer(this.container.close);

Reply:
ok